Automattic/jetpack
# Plans: Add Single Product Jetpack Backup

#### Changes proposed in this Pull Request:

This PR adds a card to the Plans page to sell Jetpack Backup.

Current development:
<img width="560" alt="image" src="https://user-images.githubusercontent.com/42627630/68362727-e48e4680-00ed-11ea-9cd5-9f981578abf7.png">

Figma design:
<img width="589" alt="image" src="https://user-images.githubusercontent.com/42627630/68362750-f7088000-00ed-11ea-85cb-b5d3605504be.png">

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
See the P2 here for the design of this PR: p1HpG7-7MK-p2 (specifically the part titled "WP Admin Desktop")
See the P2 here for the overall MT: p1HpG7-7ET-p2

#### Testing instructions:
* Go to the Plans page on a Jetpack site with a free plan.
* Verify that the Jetpack Backup card shows.
* Verify that you can toggle between realtime and daily.
* Verify that the upgrade button works.

#### Proposed changelog entry for your changes:
* Added upgrade path for Jetpack Backup to Plans page.
